common/pkgfiles/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+
+# Component: pkgfiles
+#####################################################
+# Description:
+# ------------
+#    Cmake Input File for pkgfiles
+#####################################################
+
+project( pkgfiles )
+
+set (    SRCS
+         package.cpp
+         referencedfilelist.cpp
+    )
+
+include_directories (
+        ./../../system/include
```

Comment:
personal: I much prefer the explicit fully qualified form for including paths, e.g. ${HPCC_SOURCE_DIR}/system/include

Reply:
agreed.